docs/en/observability/apm-ui/spans.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[spans]]
+=== Trace sample timeline
```

Comment:
Is this section kind of redundant with the [Trace sample timeline](https://github.com/elastic/observability-docs/pull/3723/files#diff-3e9e0e1f90bd53a1d780bad2c5c43fa0054b26f6c0bfc9b180b775c199d53ee3R129) section on the Trace samples page? There's definitely a lot more info here, not sure if this info would be to much to move into the Trace samples page. If we want to keep a separate page, should it be under **Transactions**, unless this element exists somewhere outside of the individual transactions pages.

Reply:
As discussed in our weekly sync, we're just going to "lift and shift" the content from the Kibana guide to here so I think we can refine the content in follow-up PR. (This is not the only instance of redundant content.)